src/tasks/gitea/gitea-drone-oauth.test.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai'
+import {
+  DroneSecret,
+  getGiteaAuthorizationHeaderCookies,
+  GiteaDroneError,
+  isSecretValid,
+  getCookieValue,
+} from './gitea-drone-oauth'
+import { OAuth2Application } from '@redkubes/gitea-client-node'
+import sinon from 'sinon'
+import axios from 'axios'
+
+describe('Gitea-Drone: Validate Secrets', () => {
+  let consoleLog: sinon.SinonStub
+  let sandbox: sinon.SinonSandbox
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+    consoleLog = sandbox.stub(console, 'log')
+  })
+  afterEach(() => {
+    sandbox.restore()
+  })
+  it('should successfully validate secret', () => {
```

Comment:
remove please, as I explained before we don't need this function at all

Reply:
@Morriz the purpose of this function is to validate secret content. What is your problem with this function ?

Reply:
The secret content is validated through typing as a contract. No logic is involved there. No tests should exist therefor.

Can we please move on?

Reply:
The typing is static and has nothing to do with code execution